app/dashboard/templates/profiles/profile.html

```diff
@@ -234,10 +235,32 @@
                       </div>
                     </div>
                   </div>
-                </div>
-                {% endif %}
+                </div> {% endif %}
               </div>
             {% endfor %}
+		{% else %}
+		<BR>
+		<div class="row" style="justify-content: center"><img width="240px" height="225px" src="http://kuhnchris.eu:8000/static/v2/images/tree.png" /></div><BR>
```

Comment:
`kuhnchris.eu` advertising? 😄 
Please use `{% static "v2/images...` like below 👍 .

Reply:
Hahaha weird, guess that slipped by me, sorry. ;-) Will change